src/pages/concept-list-page/concept-list-resolver.ts

```diff
@@ -1,5 +1,5 @@
 import _ from 'lodash';
-import { resolve } from 'react-resolver';
+import { resolve } from '@a-ignatov-parc/react-resolver';
```

Comment:
Seems like a shady dependency. What the change from regular react-resolver?

Reply:
Denne skjønner ikke jeg heller.

Reply:
fjernet den commit